### PR TITLE
[TECH] Report a mistake & Submit modals

### PR DESCRIPTION
**Pull request by kseniyaLine**
listening modal window only

### CODE REVIEW

File: src/components/Test/Modals/IconClose.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.close-icon-wrapper {
```
Comment (mtalstykh-exadel) [inline src/components/Test/Modals/IconClose.scss]: https://material-ui.com/components/buttons/#icon-buttons

File: src/components/Test/Modals/EssayOrSpeakingReportMistake.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```
Comment (mtalstykh-exadel) [inline src/components/Test/Modals/EssayOrSpeakingReportMistake.jsx]: combine EssayOrSpeakingReportMistake.jsx and GrammarOrListeningReportMistake.jsx. You have a lot of repeating code

File: src/pages/Test/Test.jsx [hunk 3]
```diff
@@ -1,18 +1,22 @@
-import React, { useState, useEffect } from 'react';
-import { Modal } from '@material-ui/core';
+import React, {useState, useEffect} from 'react';
+import {Modal} from '@material-ui/core';
```
Comment (mtalstykh-exadel) [inline src/pages/Test/Test.jsx]: what happend with formattig?

File: src/pages/Test/Test.jsx [hunk 4]
```diff
@@ -36,10 +40,18 @@ export const Test = () => {
...
   ];
 
+  const stepsModals = [
+    <GrammarOrListeningReportMistake key='0' tasks={grammarTasks} level={'A1'} module={'Grammar'}/>,
+    <GrammarOrListeningReportMistake key='1' tasks={listeningTasks} level={'A1'} module={'Listening'}/>,
+    <EssayOrSpeakingReportMistake key='2' level={'A1'} topic={'About Myself'} module={'Essay'}/>,
+    <EssayOrSpeakingReportMistake key='3' level={'A1'} topic={'About Myself'} module={'Speaking'}/>,
+  ];
+
+
```
Comment (mtalstykh-exadel) [inline src/pages/Test/Test.jsx]: again a problem with the linter

**Review by mtalstykh-exadel — COMMENTED** [review]
and change folder structure

File: src/components/Test/ReportAMistakeModal/ReportAMistakeModal.jsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+import './ReportAMistakeModal.scss';
+import CloseIcon from '@material-ui/icons/Close';
+import { Trans } from '@lingui/macro';
+
+export const ReportAMistakeModal = ({ tasks, topic, level, module, handleClose }) => {
+  let count = 0;
+  let HTMLCodeForStep;
+  if (module === 'Grammar' || module === 'Listening') {
+    HTMLCodeForStep =
+      <>
```
Comment (sadoffToxir) [inline src/components/Test/ReportAMistakeModal/ReportAMistakeModal.jsx:14]: you can remove a Fragment here

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Pull request merged**